### PR TITLE
review modules docs - batch(tuvwy)

### PR DESCRIPTION
**Pull request by russoz**
##### ISSUE TYPE

- Docs Pull Request

### CODE REVIEW

File: lib/ansible/modules/user.py [hunk 1]
```diff
@@ -204,13 +204,13 @@
...
             - This option does not disable the user, only lock the password.
-            - This must be set to V(False) in order to unlock a currently locked password. The absence of this parameter will not unlock a password.
+            - This must be set to V(false) in order to unlock a currently locked password. The absence of this parameter will not unlock a password.
             - Currently supported on Linux, FreeBSD, DragonFlyBSD, NetBSD, OpenBSD.
         type: bool
         version_added: "2.6"
     local:
         description:
-            - Forces the use of "local" command alternatives on platforms that implement it.
+            - Forces the use of I(local) command alternatives on platforms that implement it.
```
Comment (s-hertel) [inline lib/ansible/modules/user.py]: We actually should swap out `local` for `libuser` and deprecate the `local` alias, see https://github.com/ansible/ansible/issues/76376. I'm not sure I see the benefit of using italic.

Comment (bcoca) [reply lib/ansible/modules/user.py]: It is a bit more complex as sometimes withi `local` we edit `/etc/passwd` directly.

Also, I found out that  'module option deprecation' was not an allowed feature in modules docs,
but the following enables it #82593, though we don't have a 'common way' of deprecating aliases.

Comment (russoz) [reply lib/ansible/modules/user.py]: Hi @s-hertel there was nothing very objective about the italic there. As it seems we do not have any specific recommendation towards that so it is based on subjectivity. I am reverting that change, based on that fact. Thanks for commenting on it.

**Review by s-hertel — APPROVED** [review]
lgtm

**Comment by s-hertel** [comment]
@russoz Thanks for the PRs!

**Pull request merged**